lib/Page.js

```diff
@@ -118,7 +165,9 @@ Page.prototype.generate = function (builtFiles) {
         this.collectIncludedFiles(markbinder.getBoilerplateIncludeSrc());
         this.collectIncludedFiles(markbinder.getMissingIncludeSrc());
       })
-      .then(resolve)
+      .then(() => {
+        resolve(this.frontMatter);
+      })
```

Comment:
This change is unnecessary. We have reference to the `Page`s in `this.pageModels` in **Site.js**.